amy/autoemails/fixtures/templates_triggers.json

```diff
@@ -303,5 +303,33 @@
             "action": "consent-required",
             "template": 11
         }
+    },
+    {
+        "model": "autoemails.trigger",
+        "pk": 11,
+        "fields": {
+            "active": true,
+            "created_at": "2020-08-15T14:30:16.081Z",
+            "last_updated_at": "2020-08-15T14:30:16.081Z",
+            "action": "profile-update",
+            "template": 12
+        }
+    },
+    {
+        "model": "autoemails.emailtemplate",
+        "pk": 12,
+        "fields": {
+            "active": true,
+            "created_at": "2020-09-14T20:35:29.650Z",
+            "last_updated_at": "2020-09-14T20:35:29.650Z",
+            "slug": "profile-update",
+            "subject": "Profile Update Reminder",
+            "to_header": "{{ request.email }}",
+            "from_header": "amy-no-reply@carpentries.org",
+            "cc_header": "",
+            "bcc_header": "",
+            "reply_to_header": "",
+            "body_template": "Please [sign in](https://carpentries.org/) and update your profile information."
```

Comment:
@fmichonneau this is just barebones what I have so far. I assume we're going to want to say something more personalized.